.travis.yml

```
@@ -0,0 +1,26 @@
+language: generic
+
+services:
+  - docker
+
+cache:
+  directories:
+    - $HOME/.ccache
+
+git:
+  quiet: true
+
+env:
+  global:
+    - ROS_REPO=ros
+    - CCACHE_DIR=$HOME/.ccache
+    - AFTER_SCRIPT='./test_haros_report.sh'
```

Would this not work?

```suggestion
    - AFTER_SCRIPT='catkin build --workspace=/root/downstream_ws test_pkg --verbose --no-deps --make-args haros_report'
```

---

Oh right apparently it helps to know the arguments of `catkin tools` :smile:

---

I've verified that this works in another branch - thanks for the heads up!
I have also removed the `test_haros_report.sh`.